src/main/resources/reports/product_from_sup_facility_distribution.jrxml

```diff
@@ -53,47 +53,42 @@
 		<defaultValueExpression><![CDATA[($P{orderable} ==null) ? (" ord.id is not NULL") : (" ord.id='"+$P{orderable}+"'::uuid")]]></defaultValueExpression>
 	</parameter>
 	<queryString language="SQL">
-		<![CDATA[SELECT DISTINCT
-					o.orderCode as order_number,
-					fac.name as supplying_facility_name,
-					rec_fac.name as receiving_facility_name,
-					ord.code as product_code,
-					ord.fullProductName as product_name,
-					li.orderedQuantity as shipped_quantity,
-					CASE
-						WHEN o.status = 'RECEIVED' THEN li.orderedQuantity
-						ELSE 0
-					END as received_quantity,
-					COALESCE(
-						CASE
-							WHEN o.status = 'RECEIVED' THEN li.orderedQuantity
-							ELSE NULL
-						END, 0
-					) - COALESCE(li.orderedQuantity, 0) AS quantity_difference,
-					period.name as processing_period,
-					o.createdDate as receive_date
+		<![CDATA[SELECT DISTINCT ON (shipment_li.id)
+				o.orderCode AS order_number,
+				fac.name AS supplying_facility_name,
+				rec_fac.name AS receiving_facility_name,
+				ord.code AS product_code,
+				ord.fullProductName AS product_name,
+				COALESCE(shipment_li.quantityShipped, 0) AS shipped_quantity,
+				COALESCE(pod_li.quantityaccepted, 0) AS received_quantity,
+				COALESCE(pod_li.quantityaccepted, 0) - COALESCE(shipment_li.quantityShipped, 0) AS quantity_difference,
```

Comment:
I would recommend using ABS() function to avoid printing negative numbers. Since it's `quantity difference`field I believe it should be positive quantity.